### PR TITLE
Improve ui

### PR DESCRIPTION
**Pull request by grdaneault**
Update the email message
Changed the cover to hide the login buttons
Changed version to indicate release

### CODE REVIEW

**Comment by grdaneault** [comment]
(Fixed #40 #8 / #27 )

**Comment by regdoug** [comment]
I assume this means the backend is fixed and I can try testing my
improvements?

**Comment by grdaneault** [comment]
Yup! Test away!

File: src/main/java/edu/rit/honors/gyfp/api/folder/FolderApi.java [hunk 1]
```diff
@@ -330,10 +330,12 @@ private void sendTransferRequestMessage(TransferRequest request) {
         Properties props = new Properties();
         Session session = Session.getDefaultInstance(props, null);
 
-        // TODO make this message contain useful information
-        String messageText = "Hey there! \n"
-                + "    " + request.getRequestingUser().getEmail() + " has requested that you transfer ownership of "
-                + "your files in the folder  <a href=\"https://gimmeyourfilesplease.appspot.com/#/request/" + request.getId() + "\">" + request.getId() + "</a>.";
+        String messageFormat = "<h1>Hey there, %s!</h1> \n <p><strong>%s</strong> (%s) has requested that you transfer ownership of your files on Google Drive.  "
+                + "<a href=\"https://gimmeyourfilesplease.appspot.com/#/request/%s\">Click Here</a> to view the files they have requested and manage their ownership.</p>" +
```
Comment (regdoug) [inline src/main/java/edu/rit/honors/gyfp/api/folder/FolderApi.java]: I think "and manage their ownership" should be removed. I know what it means, but it is confusing.

I think a better wording would be "click here to reply to the request"

**Comment by grdaneault** [comment]
Message has been updated

**Pull request merged**